src/main/java/org/jabref/gui/groups/GroupNodeViewModel.java

```diff
@@ -250,6 +251,36 @@ public GroupTreeNode getGroupNode() {
         return groupNode;
     }
 
+    public boolean isAllEntriesGroup() {
+        return groupNode.getGroup() instanceof AllEntriesGroup;
+    }
+
+    /**
+     * Checks if all suggested groups already exist under this group.
+     *
+     * @return true if both "Entries without linked files" and "Entries without groups" already exist, false otherwise.
+     */
+    public boolean hasSuggestedGroups() {
+        if (!isAllEntriesGroup()) {
+            return false;
+        }
+
+        boolean hasEntriesWithoutFiles = false;
+        boolean hasEntriesWithoutGroups = false;
+
+        for (GroupNodeViewModel child : getChildren()) {
+            String name = child.getDisplayName();
+            if (Localization.lang("Entries without linked files").equals(name)) {
+                hasEntriesWithoutFiles = true;
+            }
+            if (Localization.lang("Entries without groups").equals(name)) {
```

Comment:
Localization.lang should be `private static final` constant.

Moreover use `} else if ...`, because only one of the conditions can match